ovos_persona_server/__main__.py

```
@@ -1,19 +1,32 @@
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--persona", help="path to persona .json file", required=True)
-    parser.add_argument("--host", help="host", default="0.0.0.0")
-    parser.add_argument("--port", help="port to run server", default=8337)
-    args = parser.parse_args()
+def main() -> None:
+    """
+    Main function to parse arguments and start the Persona Server.
+    """
+    parser = argparse.ArgumentParser(description="OVOS Persona Server")
+    parser.add_argument("--persona", help="Path to persona .json file", default=None, type=str)
+    parser.add_argument("--host", help="Host address to bind to", default="0.0.0.0", type=str)
```

Ruff (0.14.0)

Possible binding to all interfaces

(S104)